### ISSUE TITLE
Manual Driving/Parking Controls

### ISSUE THREAD

**Issue by AdmiralKew** [SmashPhil/Vehicle-Framework#193]
I think it would be nice to have a series of commands like:
- Move forwards 1 cell
- Move backwards 1 cell
- Rotate clockwise by 45 degrees
- Rotate counterclockwise by 45 degrees

In order to simplify the process of adjusting the placement of vehicles in their parking spaces, instead of moving them back and forth constantly with the move command.
![RimWorldWin64_xksrWKIemL](https://github.com/user-attachments/assets/fa4e19a3-3dcd-44de-84ec-2392235542f9)

**Comment by SmashPhil** [comment]
There's an even better feature I added last patch. If you hold right mouse button with the vehicle selected, you can drag to set the orientation of the vehicle.  This will set the rotation of the vehicle upon arriving at the destination.

**Comment by AdmiralKew** [comment]
Yeah, I do use that feature also. It's great for angling my vehicles to put a less damaged side into the line of fire.

I just find it finicky to use with larger vehicles, like the Abrams and Bradley I have over here and have to path it backwards and forwards until it gets into the tiles I want it to occupy.

**Comment by SmashPhil** [comment]
How so? Are those 2 vehicles even-width? The drag-to-orient feature was designed to help with parking, so if it feels clunky that would be good feedback to give.

**Comment by AdmiralKew** [comment]
The Bradley is 3 tiles wide, and the Abrams 4-wide. 

I suppose that the clunkiness stems from having to move a vehicle back and forth to get it to where you want it to be if you're off by one tile. If you're off by that amount, right clicking the ground to issue a move command only brings up the right click menu on the vehicle.

So my current workaround for this is to move it back several tiles, then move it forward again with the correct vehicle facing until it actually goes where I want it to.

**Comment by SmashPhil** [comment]
Alright, I think I understand what you mean.  I don't think adding some additional controls (bloating the gizmos even more) is the right solution, but perhaps disabling the dropdown when right clicking a vehicle with a vehicle currently selected would help alleviate this.

**Comment by SmashPhil** [comment]
Fixed awhile back, can't find the exact commit (may have been with the mass of 1.6 changes) but vehicles no longer pop open float menus when you click within their own hitbox.  Parking / adjusting a few tiles out shouldn't be a problem anymore.
